build: Markdown linter flags contents of `.sphinx` dir

## Situation

The Markdown linter isn't ignoring the contents of `.sphinx`, creating false flags for users and our PR builds fail:

```
$ make lint-md
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/container/README.md:32:1: MD022: Headings should be surrounded by blank lines. [Expected: 1; Actual: 2; Above] (blanks-around-headings,blanks-around-headers)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/container/README.md:37:1: MD014: Dollar signs used before commands without showing output (commands-show-output)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/container/README.md:42:1: MD022: Headings should be surrounded by blank lines. [Expected: 1; Actual: 2; Above] (blanks-around-headings,blanks-around-headers)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/container/README.md:59:1: MD022: Headings should be surrounded by blank lines. [Expected: 1; Actual: 2; Above] (blanks-around-headings,blanks-around-headers)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/deflist/README.md:14:1: MD022: Headings should be surrounded by blank lines. [Expected: 1; Actual: 2; Above] (blanks-around-headings,blanks-around-headers)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/deflist/README.md:36:1: MD022: Headings should be surrounded by blank lines. [Expected: 1; Actual: 2; Above] (blanks-around-headings,blanks-around-headers)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/texmath/README.md:9:1: MD022: Headings should be surrounded by blank lines. [Expected: 1; Actual: 0; Below] (blanks-around-headings,blanks-around-headers)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/texmath/README.md:47:1: MD031: Fenced code blocks should be surrounded by blank lines (blanks-around-fences)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/texmath/README.md:49:1: MD031: Fenced code blocks should be surrounded by blank lines (blanks-around-fences)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/texmath/README.md:51:1: MD031: Fenced code blocks should be surrounded by blank lines (blanks-around-fences)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/texmath/README.md:59:1: MD022: Headings should be surrounded by blank lines. [Expected: 1; Actual: 0; Below] (blanks-around-headings,blanks-around-headers)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/texmath/README.md:60:1: MD031: Fenced code blocks should be surrounded by blank lines (blanks-around-fences)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/texmath/README.md:82:1: MD031: Fenced code blocks should be surrounded by blank lines (blanks-around-fences)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/texmath/README.md:83:1: MD022: Headings should be surrounded by blank lines. [Expected: 1; Actual: 0; Above] (blanks-around-headings,blanks-around-headers)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/texmath/README.md:105:1: MD022: Headings should be surrounded by blank lines. [Expected: 1; Actual: 0; Below] (blanks-around-headings,blanks-around-headers)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/texmath/README.md:108:1: MD022: Headings should be surrounded by blank lines. [Expected: 1; Actual: 0; Below] (blanks-around-headings,blanks-around-headers)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/texmath/README.md:111:1: MD022: Headings should be surrounded by blank lines. [Expected: 1; Actual: 0; Below] (blanks-around-headings,blanks-around-headers)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/texmath/README.md:114:1: MD022: Headings should be surrounded by blank lines. [Expected: 1; Actual: 0; Below] (blanks-around-headings,blanks-around-headers)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/texmath/README.md:118:1: MD022: Headings should be surrounded by blank lines. [Expected: 1; Actual: 0; Below] (blanks-around-headings,blanks-around-headers)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/texmath/README.md:121:1: MD022: Headings should be surrounded by blank lines. [Expected: 1; Actual: 0; Below] (blanks-around-headings,blanks-around-headers)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/texmath/README.md:124:1: MD022: Headings should be surrounded by blank lines. [Expected: 1; Actual: 0; Below] (blanks-around-headings,blanks-around-headers)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/texmath/README.md:127:1: MD022: Headings should be surrounded by blank lines. [Expected: 1; Actual: 0; Below] (blanks-around-headings,blanks-around-headers)
/dev/sphinx-docs-starter-pack/docs/.sphinx/venv/lib/python3.12/site-packages/mdit_py_plugins/texmath/README.md:130:1: MD022: Headings should be surrounded by blank lines. [Expected: 1; Actual: 0; Below] (blanks-around-headings,blanks-around-headers)
make: *** [Makefile:111: lint-md] Error 1
```


## Request

Fix the blocklist in the `lint-md` target.

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -3,7 +3,8 @@
 ## Upcoming
 
 * Fix exclude paths issue for pymarkdown introduced in 0.9.35 and pin version.
-* Remove scripts and references to unused HTML metrics.
+* Add `install` dependency to `pymarkdownlnt-install` make target and use VENVDIR variable.
+* Remove scripts and references o unused HTML metrics.
 * Pin myst-parser package version to 4.0 to avoid conflicts.
 
 ### Changed

--- a/docs/Makefile
+++ b/docs/Makefile
@@ -71,8 +71,8 @@ pa11y-install:
 			npm install --prefix $(SPHINXDIR) pa11y; \
 		}
 
-pymarkdownlnt-install:
-	@. $(VENV); test -d $(SPHINXDIR)/venv/lib/python*/site-packages/pymarkdown || pip install pymarkdownlnt==0.9.35
+pymarkdownlnt-install: install
+	@. $(VENV); test -d $(VENVDIR)/lib/python*/site-packages/pymarkdown || pip install pymarkdownlnt==0.9.35
 
 install: $(VENVDIR)
 

